nodes/nomos-node/src/config.rs

```diff
@@ -72,6 +74,24 @@ pub struct NetworkArgs {
     pub initial_peers: Option<Vec<Multiaddr>>,
 }
 
+#[derive(Parser, Debug, Clone)]
+pub struct MixArgs {
+    #[clap(long = "mix-addr", env = "MIX_ADDR")]
+    mix_addr: Option<Multiaddr>,
+
+    #[clap(long = "mix-node-key", env = "MIX_NODE_KEY")]
+    mix_node_key: Option<String>,
```

Comment:
nitpick: I would rather try to have something that is not a string for this. Either the raw bytes or the key type directly. Then clap probably have something to transform to that type directly in the cli. Otherwise we have to deal with errors ourselves instead of delegating to clap itself.

Reply:
I agree. I'll try it in other PRs because there are many `String`s for node keys in this file.

Reply:
Added TODO 851c7af290fc44b51179552e5064357339b15b91